### ISSUE TITLE
Add Russian and Belorussian opposition flags #StandWithUkraine

### ISSUE THREAD

**Issue by SaveTheRbtz** [behdad/region-flags#43]
Russia and Belarus are currently totalitarian states where opposition is either [driven out of the country](https://www.nytimes.com/2021/08/30/world/europe/russian-opposition-leaders-exile.html), [imprisoned](https://www.bbc.com/news/world-europe-16057045), or [outright killed](https://www.dw.com/en/boris-nemtsov-the-man-who-dared-to-criticize-vladimir-putin/a-52561085). This lack of opposition is mainly to blame for the current war with Ukraine. It would be nice if we could help anti-war and anti-government oppositions by doing a non-disruptive region-flag addition.

The white-blue-white flag has become [the symbol of the Russian anti-war protests in 2022](https://en.wikipedia.org/wiki/White-blue-white_flag).  Sadly, there is no such flag yet.  Adding new flags to Unicode is impossible (as described by [Unicode's Emoji Proposal's Flags section](https://unicode.org/emoji/proposals.html#Flags)) but, technically, Unicode's territory subdivisions can be used to add a white-blue-white flag without any changes to the Unicode standard itself.  Russia's own Novgorod territory subdivision code can be used for this purpose: [rungr](https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/territory_subdivisions.html#rungr).

Similarly, the white-red-white flag has become [the de-facto symbol of the Belorussian protests in 2020](https://en.wikipedia.org/wiki/2020%E2%80%932021_Belarusian_protests).
In the Belorussian case, the situation is a bit trickier but still manageable: Minsk Region [bymi](https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/territory_subdivisions.html#bymi) code can be used.  Alternatively, Colombia's Atlántico territory subdivision code flag can be added: [coatl](https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/territory_subdivisions.html#coatl) has a [similar flag](https://en.wikipedia.org/wiki/File:Flag_of_Atl%C3%A1ntico.svg).

**References**
"Let's add a Belarusian national flag emoji (change.org)" https://www.change.org/p/unicode-consortium-let-s-add-a-belarusian-national-flag-emoji

**Appendix A. Currently used workarounds.**
Currently, the Russian opposition uses the following replacements:
🤍💙🤍, ⬜🟦⬜, ⚪🔵⚪, or sometimes ❌.
Similarly, the Belorussian opposition is resorting to the following sequences:
🤍❤️🤍, ⬜🟥⬜, ⚪🔴⚪.

**Comment by behdad** [comment]
While I fully sympathize with the cause, it's not clear to me what is concretely being proposed here.

**Comment by SaveTheRbtz** [comment]
TL;DR can we add a [white-blue-white](https://en.wikipedia.org/wiki/File:Anti-war_flag_of_Russian_protesters_(2022).svg) and [white-red-white](https://en.wikipedia.org/wiki/Flag_of_Belarus#/media/File:Flag_of_Belarus_(1918,_1991%E2%80%931995).svg) flags to region-flags database?  This will eventually allow them to be picked up by downstream repos (incl. android Emojis)

**Comment by behdad** [comment]
The way this repo currently works it pulls the flags from Wikipedia. If the proposed flags exist in Wikipedia, then we can add the proposed region tags. That said, adding them here does not guarantee at all that they'll be picked up by Android. Flags are very sensitive topic in operating systems...

cc @shervinafshar

**Comment by SaveTheRbtz** [comment]
Ok, then adding [Colombia's Atlántico](https://en.wikipedia.org/wiki/Atl%C3%A1ntico_Department) should give Belorussians their white-red-white flag as [ISO_3166-2:CO](https://en.wikipedia.org/wiki/ISO_3166-2:CO) `CO-ATL`.  Maybe we can start here?

The white-blue-white is a trickier one... I did not yet find an ISO_3166-2 region with this (non-historic) flag...

**Comment by behdad** [comment]
> Ok, then adding [Colombia's Atlántico](https://en.wikipedia.org/wiki/Atl%C3%A1ntico_Department) should give Belorussians their white-red-white flag as [ISO_3166-2:CO](https://en.wikipedia.org/wiki/ISO_3166-2:CO) `CO-ATL`. Maybe we can start here?

Sure. Check the `data/` directory.

**Comment by behdad** [comment]
https://www.iso.org/obp/ui/#iso:code:3166:CO

**Comment by SaveTheRbtz** [comment]
Awesome, should I add all of them or can add only a single one?

**Comment by behdad** [comment]
> Awesome, should I add all of them or can add only a single one?

If you don't mind adding all, that is preferred.

**Comment by SaveTheRbtz** [comment]
What is the preferred way of handling missing flags on Wikipedia? Should I:
* Ignore them?
* Remove these regions from the data's tsv?
* Mark them in data tsv w/ a `*` and skip based on it in `regions.py`?
* Something else?

**Comment by SaveTheRbtz** [comment]
nvm, I think I've found all of the missing flags

**Comment by SaveTheRbtz** [comment]
[White-red-white flag was added](https://github.com/fonttools/region-flags/pull/47/files#diff-0a970320c51acf0cb07d2010eeb8897d75ab7559ace75c717816da048aea6634). Closing this for now while i'm looking for a white-blue/azure-white flag.
